Server/src/loaders/loadStartupProcess.ts

```diff
@@ -1,23 +1,26 @@
 import 'dotenv/config';
+import express from 'express';
 
-import bodyParser from 'body-parser'
-import { connectDB } from '../database'
 import cors from 'cors';
-import express from 'express';
-import { fileUploadRouter } from '../routes/fileUploadRouter'
+import { connectDB } from '../database';
+import { fileUploadRouter } from '../routes/fileUploadRouter';
+import { authenticationRouter } from '../routes/authenticationRouter';
+import bodyParser from 'body-parser';
 import { getDataRouter } from '../routes/queryDataRouter';
 
 /**
  * This class contains complete startup procedure of the application. These settings are loaded only once and used
  * to initialize the application. The initial connection to the database is also created here.
  */
 export class loadStartupProcess {
+  public app: express.Application
```

Comment:
why is the app public?

Reply:
This was for testing something. Will be reverted